platform-coop-toolkit/coop-library
# feat: add homepage (resolves #117)

## Description

Adds home page layout, minus personalization features (saved searches, customizeable feed, info cards).

## Steps to test

Check out branch, `npm install`, `npm run build:production`, visit home page, review changes.

## Additional information

Not applicable.

## Related issues

- Resolves #117.
